lib/connector/authorization/authorization_converter.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:logotime/connector/view_model_converter.dart';
+import 'package:logotime/redux/action/user/authorization_action.dart';
+import 'package:logotime/widget/authorization/authorization_view_model.dart';
+
+class AuthorizationConverter
+    implements ViewModelConverter<AuthorizationViewModel> {
+  Function(dynamic) dispatch;
+  final String? emailError;
+  final String? passwordError;
+
+  AuthorizationConverter({
+    required this.emailError,
+    required this.dispatch,
+    required this.passwordError,
+  });
+
+  @override
+  AuthorizationViewModel build() {
+    return AuthorizationViewModel(
+      (viewModel) => viewModel
+        ..emailError = emailError
+        ..passwordError = passwordError
+        ..onLogInPressed = (({email, password}) => dispatch(
+              LogInAction(
+                email: email!,
```

Comment:
you should check if none of params: nor email neither password is not null

Reply:
This action is "caught" by the validationMiddleware, where the email and password are checked. I guess I should rename this action, so that it is clear that the data is being validated first